app/javascript/admin/genericAdminModal.js

```diff
@@ -0,0 +1,39 @@
+const genericAdminModal = (
```

Comment:
Can we add some documentation for how this modal works and how it should be used?

We've been using [JSDoc](https://jsdoc.app/about-getting-started.html) for JS documentation; the [initializeBroadcast.js](https://github.com/forem/forem/blob/master/app/assets/javascripts/initializers/initializeBroadcast.js) file has some examples of how we've been documenting in other JS files if you want a good starting point :)

Reply:
Oh WOW, thanks for pointing this out: I didn't know we were using something for JS documentation!